# catch exception can help us continue executing the program.